### ISSUE TITLE
Make a list of planned Acts 

### ISSUE THREAD

**Issue by baconeta** [Baconeta-Studios/setting-the-stage#4]
We should collect a list of planned act locations and physical (not game or visual) style to assist with visual referencing

This links to task #3 where we need to decide if acts will be visually distinct or follow a stylistic pattern but be just unique in the elements we see and interact with instead

**Comment by SabrinaTheCookie** [comment]
Just to confirm, is an ACT synonymous with a NIGHT?

**Comment by baconeta** [comment]
My tboughts is act = stage = ~3 nights (one setting per act)

**Comment by SabrinaTheCookie** [comment]
Okay, how about this: 
Acts hold many Chapters (what we previously called a Night). Will give us more power in storytelling and we're not limited to Night Shows 😎

**Comment by baconeta** [comment]
Yup we can use any name we choose-   - we can pseudo name it 'chapters' but for design purposes we may become more flexible with it as nights or not 😇
